src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -367,6 +367,14 @@ def __init__(self):
 
     @ViewCache()
     def _trash_pool_list(self, pool_name):
+        def _get_trashed_img(img, namespace):
```

Comment:
I think maybe to [reduce cyclomatic complexity](https://github.com/ceph/ceph/pull/40063) here shouldn't it be implemented as a separate method? Just my suggestion.

Reply:
As you wish :wink: It's not used anywhere else... so I don't thought it needs to be a separate method.